docs/references/api/tables_views.rst

```diff
@@ -217,6 +217,15 @@ This allows using the ``fts`` operator on ``text`` and ``json`` types out of the
   curl --get "http://localhost:3000/people" \
     -d "my_json_column=not.phfts(english).The%20Fat%20Cats"
 
+.. important::
+
+  To ensure this operation is fast, you need to create an index on the expression:
+
+  .. code-block:: postgres
+
+    CREATE INDEX idx_people_col ON people
+    USING GIN (to_tsvector(my_text_column));
```

Comment:
```suggestion
    USING GIN (to_tsvector('french', my_text_column));
```

According [to the docs](https://www.postgresql.org/docs/current/textsearch-tables.html#TEXTSEARCH-TABLES-INDEX):
> Notice that the 2-argument version of to_tsvector is used. Only text search functions that specify a configuration name can be used in expression indexes ([Section 11.7](https://www.postgresql.org/docs/current/indexes-expressional.html)). This is because the index contents must be unaffected by [default_text_search_config](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-DEFAULT-TEXT-SEARCH-CONFIG).